src/app/legacy/containers/StoryPromo/helpers/fixtureData.js

```diff
@@ -269,7 +265,7 @@ export const mapWithMediaError = {
   },
   section: {
     name: 'کور پاڼه',
-    subType: INDEX_PAGE,
+    subType: 'IDX',
     type: 'simple',
     uri: '/pashto/front_page',
   },
```

Comment:
Just curious if these should be the `FRONT_PAGE` type? Not too familiar with these routes!

Reply:
No probs, thanks for clarifying 👍